UnleashClient/api/register.py

```diff
@@ -52,7 +52,7 @@ def register_client(
         "interval": metrics_interval,
         "platformName": python_implementation(),
         "platformVersion": python_version(),
-        "yggdrasilVersion": None,
+        "yggdrasilVersion": "0.14.0",
```

Comment:
Can you do something like `yggdrasil.__version__` ?  I think a constant/tying it back to the requirements might be better!

Reply:
Yes! I realised this last night actually and I was wondering if someone was going to spot it :laughing: Good spot, will swap it out

Reply:
Fixed in [02bb05](https://github.com/Unleash/unleash-client-python/pull/322/commits/02bb05ec4817c4d68ddcafe7102c1ddbadf97b6f)